bitnami/thanos/values.yaml

```diff
@@ -1834,6 +1847,45 @@ storegateway:
     ## @param storegateway.pdb.maxUnavailable Maximum number/percentage of pods that may be made unavailable
     ##
     maxUnavailable: ""
+  ## Sharded parameters
+  ## @param storegateway.sharded.enabled Enable sharding for Thanos Store Gateway
+  ## @param storegateway.sharded.hashPartitioning.shards Setting hashPartitioning will create multiple store statefulsets based on the number of shards specified using the hashmod of the blocks
+  ## @param storegateway.sharded.timePartitioning [array] Setting time timePartitioning will create multiple store deployments based on the number of partitions
+  ## @param storegateway.sharded.service.clusterIPs Array of cluster IPs for each Store Gateway service. Length must be the same as the number of shards
+  ## Example:
+  ## clusterIPs:
+  ##   - X.X.X.X
+  ##   - Y.Y.Y.Y
+  ## @param storegateway.sharded.service.loadBalancerIPs Array of load balancer IPs for each Store Gateway service. Length must be the same as the number of shards
+  ## Example:
+  ## loadBalancerIPs:
+  ##   - X.X.X.X
+  ##   - Y.Y.Y.Y
+  ## @param storegateway.sharded.service.http.nodePorts Array of http node ports used for Store Gateway service. Length must be the same as the number of shards
+  ## Example:
+  ## nodePorts:
+  ##   - 30001
+  ##   - 30002
+  ## @param storegateway.sharded.service.grpc.nodePorts Array of grpc node ports used for Store Gateway service. Length must be the same as the number of shards
+  ## Example:
+  ## nodePorts:
+  ##   - 30011
+  ##   - 30012
+  ##
```

Comment:
All this metadata is missing in the **README.md**, you can use our [readme generator](https://github.com/bitnami-labs/readme-generator-for-helm) tool to automatically update it based on this metadata for you.